ortools/sat/csharp/CpSolver.cs

```diff
@@ -11,12 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using System;
```

Comment:
`IDisposable` seems to comes from `System` why did you remove it ?
https://learn.microsoft.com/en-us/dotnet/api/system.idisposable?view=net-8.0